src/Endpoints/Pages.php

```diff
@@ -33,23 +33,81 @@ public function find(string $pageId): Page
     }
 
     /**
-     * @return array
-     * @throws HandlingException
+     * @return Page
      */
-    public function create(): array
+    public function createInDatabase(string $parentId, Page $page): Page
     {
-        //toDo
-        throw new HandlingException( 'Not implemented');
+        $postData = [];
+        $properties = [];
+
+        foreach ($page->getProperties() as $property) {
+            $properties[$property->getTitle()] = $property->getRawContent();
+        }
+
+        $postData["parent"] = ["database_id" => $parentId];
+        $postData["properties"] = $properties;
+
+
+        $response = $this
+            ->post(
+                $this->url(Endpoint::PAGES),
+                $postData
+            )
+            ->json();
+
+        return new Page($response);
     }
 
+        /**
+     * @return Page
+     */
+    public function createInPage(string $parentId, Page $page): Page
+    {
+        $postData = [];
+        $properties = [];
+
+        foreach ($page->getProperties() as $property) {
+            $properties[$property->getTitle()] = $property->getRawContent();
+        }
+
+        $postData["parent"] = ["page_id" => $parentId];
+        $postData["properties"] = $properties;
+
+
+        $response = $this
+            ->post(
+                $this->url(Endpoint::PAGES),
+                $postData
+            )
+            ->json();
+
+        return new Page($response);
+    }
+
+
 
     /**
      * @return array
      * @throws HandlingException
      */
-    public function updateProperties(): array
+    public function updateProperties(Page $page): Page
```

Comment:
I just started writing tests and asked myself if just `update` as method name could be already enough. 🤔  @johguentner

Reply:
Yeah, I thought too, however we need to make sure it's clear, that >only< the properties are updated. 
The notion-docs indicate even for the future (at least with this endpoint) only properties are modified -> https://developers.notion.com/reference/patch-page

However if you think "update" for page is not confusing regarding the blocks, I wouldn't mind if we change it to "update".
(Probably blocks will not be modified with the page-endpoint, but only with the block-endpoint)

Reply:
Just checked the official ts notion-sdk from notion.
They called it "update", too:
https://github.com/makenotion/notion-sdk-js/blob/main/src/api-endpoints.ts#L215

So lets do that as well 😊